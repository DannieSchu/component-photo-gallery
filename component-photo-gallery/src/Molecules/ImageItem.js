import React, { Component } from 'react';

export default class ImageItem extends Component {
    render() {
        return(
            <div className = "image-item">
                <h3>{this.props.image.title}</h3>
                <img 
                    src = {this.props.image.url} 
                    alt = "unicorn" />
                <p>{this.props.image.description}</p>
            </div>
        )
    }
}