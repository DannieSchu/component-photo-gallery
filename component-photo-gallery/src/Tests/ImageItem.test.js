import React from 'react'
import renderer from 'react-test-renderer';
import ImageItem from '../Molecules/ImageItem';


test('renders ImageItem.js correctly', () => {
    const item = {}
    const tree = renderer
      .create(<ImageItem image={item}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });