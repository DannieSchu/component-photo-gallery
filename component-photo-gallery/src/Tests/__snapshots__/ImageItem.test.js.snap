// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageItem.js correctly 1`] = `
<li
  className="flex-column"
>
  <h3 />
  <div
    className="image-container"
  >
    <img />
  </div>
  <p />
</li>
`;
